modules/user/user.controller.js

```diff
@@ -1,66 +1,109 @@
 const express = require("express");
 var router = express.Router(); // - /v1/user/
-
-const filePath = "./database/users.json";
 const jsonParser = express.json();
-const fs = require("fs");
-
-//получение всех пользователей
-router.get("/", function (req, res) {
-    fs.readFile(filePath, function (err, content) {
-        if (err) {
-            res.statusCode = 500;
-            res.end("Server error");
-            return;
-        }
+const { isEmpty, getUser } = require("../../utils.js");
+const userService = require("./user.service");
 
-        const users = JSON.parse(content);
+router.get("/", async function (req, res) {
+    try {
+        let users = await userService.readAllUsers();
         res.send(users);
-    });
+    } catch (err) {
+        res.statusCode = 500;
+        res.end("Server error");
+    }
 });
 
-//получение конкретного пользователя
-router.get("/:id", function (req, res) {
-    const id = req.params.id;
-    let userById = null;
+router.get("/:id", async function (req, res) {
+    try {
+        const id = req.params.id;
+
+        let user = await userService.readOneUser(id);
 
-    fs.readFile(filePath, function (err, content) {
-        if (err) {
+        res.send(user);
+    } catch (err) {
+        if (err.message == 404) {
+            res.statusCode = 404;
+            res.end("User not found");
```

Comment:
it's better to store all error messages in one place

- [ ] move all error message to one place like `messages.js`